sktime/forecasting/tests/test_reconcile.py

```diff
@@ -122,3 +122,40 @@ def test_reconcilerforecaster_exog(n_columns):
     estimator_instance.fit(y=y_train, X=X_train, fh=fh)
     estimator_instance.predict(X=X_test)
     estimator_instance.update(y=y_test, X=X_test)
+
+
+def test_reconcilerforecaster_return_totals(return_totals=False):
```

Comment:
Question: why does this have `return_totals` as an argument?

We should maybe also test `return_totals=True`, and there in the case of `hierarchy_levels=(m, n)` the total length should be `len(y_pred) = len(y_test) + 1 + 4 + 4*3

Reply:
i think this case will work out - for hierarchy_levels=(m, n) `len(y_pred) = len(y_test) + (1 + m) * 2`